test/serializer/optimized-functions/Issue2266Regression.js

```
@@ -0,0 +1,12 @@
+// expected Warning:
```

I'm not sure it makes sense to a test writer that if they write `// expected <Severity>:` without any error codes that it gets silently ignored.

Could we make it an invariant or user-facing error if the list of expected error codes is empty?

---

That is exactly what I wanted here: Make sure there is no Warning. (Because before my change, there was a non-sensical warning mentioning an internal Prepack concept that shouldn't raise a warning.)

---

And I changed the test runner to actually enforce it that way.